Practice Areas/Engineering/code_review_guidelines.md

```diff
@@ -12,12 +12,18 @@ Before sending your code to a VSP team for review, your code should meet all of
 ## Guidelines
 
 * **Review lifecycle**
-    * Before sending code to VSP team, have it reviewed by your team (see **First review** below).
-    * An engineer from the VSP will aim to review within 24 hours (business days) and each additional round of changes will be reviewed within 24 hours.  If you need it faster, escalate to Leah and Patrick.
+    * Before sending code to VSP team have it reviewed by your team (see **First review** below) using a [Draft Pull Request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests).  
+      * This prevents [code owners](https://help.github.com/en/articles/about-code-owners) from being notified before pull request is ready for review.
+    * After having your pull request reviewed by your team you can mark your pull request as ready for review.
```

Comment:
Might be worth calling out how to do this. Is there a button to graduate a draft PR to a full-fledged PR?

Reply:
There is, 
<img width="680" alt="Screen Shot 2019-08-07 at 2 19 41 PM" src="https://user-images.githubusercontent.com/1094999/62647383-6b305d80-b91e-11e9-964b-7ca41a85bc5b.png">

Reply:
updating section to indicate this

Reply:
also added link to https://github.blog/2019-02-14-introducing-draft-pull-requests/